app/controllers/api/base_controller.rb

```diff
@@ -233,6 +234,17 @@ def add_version_header
       response.headers["Foreman_api_version"]= api_version
     end
 
+    def add_taxonomies_header
+      if SETTINGS[:organizations_enabled]
+        current_org = "#{Organization.current.id}; #{Organization.current.name}" if Organization.current
+        response.headers["Current_organization"] = current_org || '; ANY'
```

Comment:
How about `Foreman_current_organization`? The name clash is fairly minimal here, but I'd prefer to keep the `Foreman_` prefix here for the sake of consistency.